multiplayer/src/components/HeaderBar.tsx

```diff
@@ -0,0 +1,147 @@
+/// <reference path="../../../localtypings/react.d.ts" />
+
+// TODO multiplayer : Reduce duplication with skillmap HeaderBar.tsx (may require enabling tailwind css there or removing it here?)
+
+import { Button } from "../../../react-common/components/controls/Button";
+import { MenuBar } from "../../../react-common/components/controls/MenuBar";
+import { MenuDropdown, MenuItem } from "../../../react-common/components/controls/MenuDropdown";
+
+interface HeaderBarProps {
+    showReportAbuse?: boolean;
```

Comment:
showReportAbuse unused prop?

Reply:
I think this will pop a modal dialog eventually. Another argument to manage modals the way skillmap does.